src/main/java/io/debezium/connector/vitess/VitessType.java

```diff
@@ -94,13 +94,14 @@ public static VitessType resolve(Query.Field field) {
                 return new VitessType(type, Types.BIGINT, resolveEnumAndSetValues(field.getColumnType()));
             case "UINT32":
             case "INT64":
-            case "UINT64":
                 return new VitessType(type, Types.BIGINT);
+            case "UINT64":
```

Comment:
I think we also need to update the [resolver logic](https://github.com/debezium/debezium-connector-vitess/blob/9a87a5ce642d30f2e7f109755ad78916c4ca1227/src/main/java/io/debezium/connector/vitess/connection/ReplicationMessageColumnValueResolver.java#L28-L30) accordingly for this change

Reply:
> Looks like kafka connect also suggests to convert uint64 to bytes instead of throwing exceptions
Before this PR, it doesn't throw exceptions upon overflow; it silently become -1, which is a problem.

> I think we also need to update the resolver logic accordingly for this change
Good catch. Will do